driver/statement.cpp

```diff
@@ -307,7 +307,7 @@ std::string Statement::buildFinalQuery(const std::vector<ParamBindingInfo>& para
         std::string param_type;
 
         if (param_bindings.size() <= i) {
-            param_type = "Nullable(Nothing)";
+            param_type = "Nullable(String)";
```

Comment:
I don't get this one. 

Also check comment
https://github.com/ClickHouse/clickhouse-odbc/blob/3b61ca7b2050f042da1d7faabb3d23b6627a4200/driver/statement.cpp#L435

Reply:
Updated.